docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md

```diff
@@ -281,137 +146,318 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
+```
+
+New:
+
+```ts
+getConfigurations(): OpenIdConfiguration[] { }
+
+
+getConfiguration(configId?: string): OpenIdConfiguration { }
+```
+
+used with
+
+```ts
+const configuration = this.oidcSecurityService.getConfiguration();
+// or
+const configuration = this.oidcSecurityService.getConfiguration('configId');
+```
+
+### `getToken()` renamed to `getAccessToken()`
+
+Old
+
+```ts
+const accessToken = this.oidcSecurityService.getToken();
+```
+
+New
+
+```ts
+const accessToken = this.oidcSecurityService.getAccessToken();
+```
+
+### `checkAuth()` returning `LoginResponse` instead of boolean
+
+The `checkAuth()` method is now not only returning if you are authenticated or not but instead a complete object containing:
+
+```json
+{
+  isAuthenticated: boolean;
+  userData: any;
+  accessToken: string;
+  idToken: string;
+  configId: string;
+  errorMessage?: string;
+}
+```
+
+Old:
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe((isAuthenticated) => {});
+```
+
+New:
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated }) => {});
+```
+
+Or
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated, userData, accessToken, idToken, configId }) => {
+  // ...use data
+});
+```
+
+### `checkAuthIncludingServer()` returning `LoginResponse` instead of boolean
+
+See [`checkAuth()` returning `LoginResponse` instead of boolean](#checkauth-returning-loginresponse-instead-of-boolean).
+
+Same applies for `checkAuthIncludingServer()`
+
+### `forceRefreshSession()` returning `LoginResponse` instead of TokenResponse
+
+See [`checkAuth()` returning `LoginResponse` instead of boolean](#checkauth-returning-loginresponse-instead-of-boolean).
+
+Same return value applies for `forceRefreshSession()`.
+
+Old:
+
+```ts
+this.oidcSecurityService.forceRefreshSession().subscribe((isAuthenticated) => {});
+```
+
+New:
+
+```ts
+this.oidcSecurityService.checkAuth().subscribe(({ isAuthenticated }) => {});
+```
+
+### `authorize(...)` has new `configId` as first parameter
+
+Because multiple configs have been introduce with V12, now the `configId` is the first parameter.
+
+Old:
+
+```ts
+const authOptions = {...}
+this.oidcSecurityService.authorize(authOptions)
+```
+
+New:
+
+```ts
+const authOptions = {...}
+const configIdOrNull = ./*...*/;
+this.oidcSecurityService.authorize(configIdOrNull, authOptions)
+```
+
+### `logoffAndRevokeTokens(...)` has new `configId` as first parameter
+
+Because multiple configs have been introduce with V12, now the `configId` is the first parameter.
+
+Old:
+
+```ts
+const authOptions = {...}
+this.oidcSecurityService.logoffAndRevokeTokens(authOptions)
+```
+
+New:
+
+```ts
+const authOptions = {...}
+const configIdOrNull = ./*...*/;
+this.oidcSecurityService.logoffAndRevokeTokens(configIdOrNull, authOptions)
+```
+
+### `logoff(...)` has new parameters
+
+Because multiple configs have been introduce with V12, now the `configId` is the first parameter and `AuthOptions` as a second.

```

Comment:
Minor rewording for clarity:

Because V12 introduces multiple configs, the first parameter is now `configId` and the second parameter is `AuthOptions`.